Dart/src/com/jetbrains/lang/dart/ide/actions/DartEditorNotificationsProvider.java

```diff
@@ -105,11 +105,14 @@ public EditorNotificationPanel createNotificationPanel(@NotNull final VirtualFil
   private static class PubActionsPanel extends EditorNotificationPanel {
     private PubActionsPanel() {
       super(EditorColors.GUTTER_BACKGROUND);
-      myLinksPanel.add(new JLabel("Pub actions:"));
+
+      icon(DartIcons.Dart_16);
+      text("Pub actions");
+
```

Comment:
I asked 2 more colleagues and we decided to keep label and links close to each other.